test/xpu_api/random/statistics_tests/bernoulli_distribution_test.pass.cpp

```diff
@@ -116,7 +116,7 @@ test_portion(sycl::queue& queue, double p, int nsamples, unsigned int part)
 
                 sycl::vec<bool, num_elems> res = distr(engine, part);
                 for (int i = 0; i < n_elems; ++i)
-                    acc.get_pointer()[idx.get_linear_id() * n_elems + i] = res[i];
+                    __dpl_sycl::__get_accessor_ptr(acc)[idx.get_linear_id() * n_elems + i] = res[i];

```

Comment:
Can we replace it with:
```suggestion
                    acc[idx.get_linear_id() * n_elems + i] = res[i];
```